lib/fxpay/pay.js

```diff
@@ -94,6 +93,15 @@
       // load it into the freshly created popup window.
       paymentWindow.location = payUrl;
 
+      // This interval covers closure of the popup
+      // whilst with the provider.
```

Comment:
might be good to make this explicit: "...whilst with the provider since they won't send us postMessage data."